buildspec.yml

```diff
@@ -4,6 +4,8 @@ env:
   shell: bash
   variables:
     AWS_DEFAULT_REGION: eu-central-1
+    AWS_REGION: eu-central-1
```

Comment:
Why those two variables ?

Reply:
That was for testing with the local build bash script, it was complaining that no region was set.